src/pages/stop/stop.component.ts

```diff
@@ -120,6 +121,12 @@ export class StopComponent {
       }).catch(err => {
         console.error(`Error retrieving refresh time: ${err}`);
       });
+      this.storage.get('departureSort').then(storedSort => {
```

Comment:
If there's no departureSort in storage because someone didn't sort it, is it OK?

Reply:
Yep!